README.md

```diff
@@ -16,19 +16,32 @@ using QE tiered testing.
   1. Edit the test variables file to include information required for this test run.
      You will need to add info required for test execution:
      * URLs based on the test case you are executing.<br>
-        ** OpenShift Console.<br>
-        ** Open Data Hub Dashboard.<br>
-        ** JupyterHub.<br>
+        *   OpenShift Console.<br>
+        *   Open Data Hub Dashboard.<br>
+        *   JupyterHub.<br>
      * Test user credentials.
      * Browser webdriver to use for testing.
-  1. Run this script that will create the virtual environment, install the required packages and kickoff the Robot test suite
-    ```bash
-    sh run_robot_test.sh
-    ```
-    This script is a wrapper for creating the python virtual environment and running the Robot Framework CLI.  You can run any of the test cases by creating the python virual environment, install the packages in requirements.txt and running the `robot` command directly
+
+  1. Run this script that will create the virtual environment, install the required packages and kickoff the Robot test suite.
+  ```bash
+     # running all the tests 
+     sh run_robot_test.sh 
+
+     # running Smoke test suite via tag
+     sh run_robot_test.sh --include Smoke 
+
+     # running a specific test via tag
+     sh run_robot_test.sh --include ODS-XYZ 
```

Comment:
As `TEST_VARIABLES_FILE=ods_ci/test-variables.yml` in run_robot_test.sh, the command that works for me **from the root of the project** is this (same for the previous commands):
 
`sh ods_ci/run_robot_test.sh --include ODS-233  --skip-oclogin`

Reply:
true! fixed. Thanks